pom.xml

```diff
@@ -3150,23 +3146,7 @@
 
     <profile>
       <id>hive-1.2</id>
-      <properties>
-        <hive.group>org.spark-project.hive</hive.group>
-        <hive.classifier></hive.classifier>
-        <!-- Version used in Maven Hive dependency -->
-        <hive.version>1.2.1.spark2</hive.version>
-        <!-- Version used for internal directory structure -->
-        <hive.version.short>1.2</hive.version.short>
-        <hive.parquet.scope>${hive.deps.scope}</hive.parquet.scope>
-        <hive.storage.version>2.6.0</hive.storage.version>
-        <hive.storage.scope>provided</hive.storage.scope>
-        <hive.common.scope>provided</hive.common.scope>
-        <hive.llap.scope>provided</hive.llap.scope>
-        <hive.serde.scope>provided</hive.serde.scope>
-        <hive.shims.scope>provided</hive.shims.scope>
-        <orc.classifier>nohive</orc.classifier>
-        <datanucleus-core.version>3.2.10</datanucleus-core.version>
-      </properties>
+      <!-- Exists only for backward compatibility. No-op. -->
```

Comment:
So this profile is still useful?

Reply:
No, it's just that you get an error if you specify `-Phive-1.2` if it's removed entirely. Now, maybe that's a good thing? previously for Scala profiles I had left them in because, for example `-Pscala-2.12` was not the default before. When it became the default, that would have caused an error even though it was already set up for 2.12. But this case is probably different: someone selecting Hive 1.x support should see an error, probably? in which case this should be removed.

Reply:
Yeah, this sounds more correct.

Reply:
Got it. I'll remove this and raise error explicitly, @viirya and @srowen . :)